etc/init.d/zfs-mount.in

```diff
@@ -57,7 +57,7 @@ do_depend()
 	# bootmisc will log to /var which may be a different zfs than root.
 	before bootmisc logger
 
-	after procfs zfs-import sysfs procps
```

Comment:
See the end of https://github.com/zfsonlinux/zfs/issues/3513#issuecomment-114600082 and the beginning of https://github.com/zfsonlinux/zfs/issues/3513#issuecomment-114604681.  The apparent issue is that there is a dependency loop with `zfs-mount` depending on `localmount` which depends on `procfs` which depends on `localmount`.  I read the `procfs` service script and saw that it only really took care of mounting things like `binfmt_misc` and `usbfs`, neither of which are required for `zfs-mount` so I removed the dependency and that solved the dependency loop.  OpenRC itself seems to take care of mounting `/proc` (and thus making `/proc/mounts` available) before starting any other services.